src/main/java/gg/solarmc/kitpvp/kill/damage/DamageMap.java

```diff
@@ -1,13 +1,13 @@
-package gg.solar.kitpvp;
+package gg.solarmc.kitpvp.kill.damage;
 
-import me.aurium.beetle.core.config.DataHolder;
+import gg.solarmc.kitpvp.KitpvpPlugin;
```

Comment:
This class should not retain Player objects in a Map beyond when the players have quit. Also, the Map should not grow indefinitely - use a cache if necessary. Moreover trackDamage can use getHolder

Reply:
resolved